plugins/builddep.py

```diff
@@ -135,13 +135,15 @@ def _rpm_dep2reldep_str(rpm_dep):
         return rpm_dep.DNEVR()[2:]
 
     def _install(self, reldep_str):
-        try:
-            self.base.install(reldep_str)
-        except dnf.exceptions.MarkingError:
+        sltr = dnf.selector.Selector(self.base.sack)
+        sltr = sltr.set(provides=reldep_str)
```

Comment:
this seems clean but shouldn't it consider `sltr.set(file=reldep_str)` if no such provide exists?